posts/2025-02-17_theres_a_pharmaverse_package_for_that/managing-clinical-trial-data.qmd

```diff
@@ -0,0 +1,323 @@
+
+## **Key Stages of the Clinical Trial Process**
+
+Managing clinical trial data involves multiple stages, each with its own challenges. pharmaverse provides a range of packages for different parts of the process, sometimes even more than one option for the same task, so you can choose what works best for your needs rather than sticking to a single approach
+
+![](images/clinical-trial-process.png)
+
+1.  **Collection** – Initial data handling using:
+
+    -   [**tidyverse**](https://www.tidyverse.org/) for general data manipulation.
+
+    -   [**diffdf**](https://cran.r-project.org/web/packages/diffdf/vignettes/diffdf-basic.html) for tracking differences in datasets.
+
+    -   [**metatools**](https://pharmaverse.github.io/metatools/) for metadata management and transformation.
+
+2.  **Tabulation** – Structuring and validating data with:
+
+    -   [**thevalidatoR**](https://github.com/marketplace/actions/r-package-validation-report) for data validation checks.
+
+    -   [**covtracer**](https://github.com/Genentech/covtracer) for ensuring code traceability and reproducibility.
+
+3.  **Analysis Ready** – Preparing data for regulatory-grade analytics:
+
+    -   [**admiral**](https://github.com/pharmaverse/admiral) for standardized data derivations.
+
+    -   [**metacore**](https://cran.r-project.org/web/packages/metacore/index.html) for metadata-driven structures.
+
+    -   [**dplyr**](https://dplyr.tidyverse.org/) (part of the tidyverse) for data manipulation
+
+4.  **Analysis Results** – Incorporating visualization and modeling tools:
+
+    -   [**ggplot2**](https://ggplot2.tidyverse.org/) (part of the tidyverse) for advanced data visualization.
+
+    -   [**visR**](https://github.com/openpharma/visR) for survival analysis and Kaplan-Meier plots.
+
+    -   [**pharmaRTF**](https://cran.r-project.org/web/packages/pharmaRTF/index.html) for creating submission-ready reports.
+
+    -   [**rtables**](https://github.com/insightsengineering/rtables) for structured table outputs
+
+    -   [**Tplyr**](https://github.com/atorus-research/Tplyr) for clinical summary tables
+
+    -   The pharmaverse offers multiple table-making packages, including [rtables](https://insightsengineering.github.io/rtables/v0.6.10/articles/rtables.html), [chevron](https://insightsengineering.github.io/chevron/latest-tag/), [pharmaRTF](https://atorus-research.github.io/pharmaRTF/), [gtsummary](https://www.danieldsjoberg.com/gtsummary/), [cards](https://insightsengineering.github.io/cards/latest-tag/), [tfrmt](https://gsk-biostatistics.github.io/tfrmt/), [tidytlg](https://pharmaverse.github.io/tidytlg/main/), and [cardinal](https://pharmaverse.github.io/cardinal/). You can find more tools on the [TLGs page](https://pharmaverse.org/e2eclinical/tlg/). Each serves different needs, and it is up to the programmer to choose the best fit.
```

Comment:
i wouldn't mention cardinal here as its an examples book to show how to use different table packages for the FDA templates - its not a table-making package itself